lib/internal/main/repl.js

```diff
@@ -13,10 +13,11 @@ const {
 
 prepareMainThreadExecution();
 
-// --entry-type flag not supported in REPL
-if (require('internal/options').getOptionValue('--entry-type')) {
-  console.error('Cannot specify --entry-type for REPL');
-  process.exit(1);
+// --input-type flag not supported in REPL
+if (require('internal/options').getOptionValue('--input-type')) {
+  const { ERR_INPUT_TYPE_NOT_ALLOWED } = require('internal/errors').codes;
+  // Cannot specify --input-type for REPL.
+  throw new ERR_INPUT_TYPE_NOT_ALLOWED();
```

Comment:
Suggestion to avoid printing a stack trace:

```suggestion
  const error = new ERR_INPUT_TYPE_NOT_ALLOWED();
  console.error(error.message);
  process.exit(1);
```